src/main/java/com/mraof/minestuck/client/particle/MSParticleType.java

```diff
@@ -0,0 +1,32 @@
+package com.mraof.minestuck.client.particle;
+
+import com.mraof.minestuck.Minestuck;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.particle.ParticleEngine;
+import net.minecraft.core.particles.ParticleType;
+import net.minecraft.core.particles.SimpleParticleType;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+import net.minecraftforge.client.event.RegisterParticleProvidersEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.registries.DeferredRegister;
+import net.minecraftforge.registries.ForgeRegistries;
+import net.minecraftforge.registries.RegistryObject;
+
+@Mod.EventBusSubscriber(modid = Minestuck.MOD_ID, bus = Mod.EventBusSubscriber.Bus.MOD, value = Dist.CLIENT)
+public class MSParticleType
+{
+	public static final DeferredRegister<ParticleType<?>> REGISTER = DeferredRegister.create(ForgeRegistries.PARTICLE_TYPES, Minestuck.MOD_ID);
+	
+	public static final RegistryObject<SimpleParticleType> TRANSPORTALIZER = REGISTER.register("transportalizer", () -> new SimpleParticleType(false));
+	
+	@OnlyIn(Dist.CLIENT)
+	@SubscribeEvent
+	public static void registerFactories(RegisterParticleProvidersEvent event)
+	{
+		ParticleEngine particles = Minecraft.getInstance().particleEngine;
+		
+		particles.register(MSParticleType.TRANSPORTALIZER.get(), TransportalizerParticle.Provider::new);
```

Comment:
There's an `event.register()` function that I think you're meant to be using.

Reply:
Changed to use event.register()